src/stylesheets/components/_accordions.scss

```diff
@@ -19,24 +15,30 @@ $v-padding: $spacing-md-small;
 // scss-lint:disable PropertyCount
 @mixin accordion-button-styles {
   @include button-unstyled;
-  background-color: $color-gray-lightest;
+  @include u-font-family('sans');
+  @include u-padding(
```

Comment:
Sorry, the `u-padding()` mixin cant take multiple arguments in this form (yet?) — use a conventional padding rule (or rules) instead: